src/Package/MSTest.Sdk/Sdk/Runner/Runner.targets.template

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Import Project="$(MSBuildThisFileDirectory)ClassicEngine.targets" Condition=" '$(EngineName)' == 'Classic' "/>
+  <Import Project="$(MSBuildThisFileDirectory)ClassicEngine.targets" Condition=" '$(TestEngineName)' == 'Classic' "/>
 
-  <Import Project="$(MSBuildThisFileDirectory)TestAnywhereEngine.targets" Condition=" '$(EngineName)' == 'TestAnywhere' "/>
+  <Import Project="$(MSBuildThisFileDirectory)TestingPlatform.targets" Condition=" '$(TestEngineName)' == 'TestAnywhere' "/>
```

Comment:
I would avoid leaking `TestAnywhere` to public. Can we use something else? I am not sure if this engine relates to the new one supporting reflection + reflection-free or linked to the runner.

Reply:
This engine should be the one allow us to run anywhere if it's using or not reflection should not be a detail users should know imo.
I'm fine with whatever name I had already some complain about it.
Here testanywhere is only a word it's not a product or something else, I know we're biased by internal usage but outside means "run tests anywhere" and wasn't meant to link with our internal codename.

@nohwnd @pavelhorak do you have some idea to suggest